public/locales/cs/translation.json

```diff
@@ -91,7 +92,8 @@
     "ataSystem": "ATA systém",
     "partNumber": "Číslo součásti",
     "stock": "Zásoba",
-    "quantity": "Množství"
+    "quantity": "Množství",
+    "schematicDescription": "Schematický popis"
```

Comment:
"schematicDescription": "Schematický popis"
->
"schematicDesignation": "Schématické označení"


Is there any point to do schematicDesignation -> schematicDescription ? To me the terms means little bit different thing.

Reply:
Changed.